packages/backend/src/Controllers/Election/getElectionsController.ts

```diff
@@ -20,7 +20,7 @@ const getElections = async (req: IElectionRequest, res: Response, next: NextFunc
 
     /////////// ELECTIONS WE OWN ////////////////
     var elections_as_official = null;
-    if(email !== '' || id !== ''){ 
```

Comment:
Previously the /manage page would show all the elections you've made, even if you don't have an account.

This created an awkward window in the login flow where you see your old elections before seeing your new ones. Since the /manage people is only accessible outside of login by direct link, it seemed fine to disable the election list when you're not logged in.